digiwf-apps/packages/apps/digiwf-tasklist/src/views/TaskDetail.vue

```diff
@@ -225,7 +225,7 @@ export default class TaskDetail extends SaveLeaveMixin {
       if (!!data) {
         this.task = data.task;
         this.model = data.model;
-        this.followUpDate = data.followUpDate;
+        this.followUpDate = data.followUpDate?.replaceAll(".", "-"); // only for usage in v-date-picker
```

Comment:
Den verstehe ich hier nicht ganz. Warum ist das notwendig?

Reply:
Wir haben das Problem, dass v-data-picker nur ISO 8601 strings akzeptiert. 
Wir transformieren aber das format schon für den Nutzer formatiert in der Middleware. 

Alternative zu der markierten Zeile wäre aus meiner Sicht 2 properties zu machen: 
1. followUpDate: ISO 8601 Format
2. followUpDateFormatted: i18n formatiert

Dafür hätte ich es aber für Engine und Tasklist machen müssen. das war mir für den Fix jetzt zu viel. 
Würde aus meiner Sinn nach dem Entfernen der ganzen  Engine Requestcalls Sinn machen.

Reply:
Ah okay, ja passt als Workaround